python/metatensor_torch/tests/block.py

```diff
@@ -281,7 +281,7 @@ def test_to():
     check_dtype(block, torch.float32)
     check_dtype(block.gradient("g"), torch.float32)
 
-    converted = block.to(dtype=torch.float64)
+    converted = block.to(dtype=torch.float64, non_blocking=True)
```

Comment:
Maybe something as simple as

```suggestion
    # smoke test for non_blocking, see gh-920
    converted = block.to(dtype=torch.float64, non_blocking=True)
```

Reply:
this test is also properly checking the rest of `to`. But I'll find a way to indicate this!

Reply:
Better now?

Reply:
Yup, excellent.